digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/dataimport/CsvDataImporter.scala

```diff
@@ -126,6 +125,36 @@ class TrafficSignCsvImporter extends CsvDataImporterOperations {
     }
   }
 
+  def getRoadLinkByBearing(assetBearing: Option[Int], assetValidityDirection: Option[Int], assetCoordinates: Point, roadLinks: Seq[RoadLink]): Seq[RoadLink] = {
```

Comment:
should this method be here or on a more general service? we might need it for future assets.